test/client/client_fixtures.js

```diff
@@ -206,6 +206,66 @@ export default {
       created_at: '2017-08-15T21:35:30.321Z',
       updated_at: '2017-08-15T21:35:30.321Z'
     }
+  },
+  scriptViewFixtures: {
+    defaultScript: {
```

Comment:
Might be nice to import the default script obj from the source file (reducer), rather than hard code this here.